kloppy/infra/serializers/tracking/epts/serializer.py

```diff
@@ -120,8 +120,8 @@ def deserialize(
                     raw_data=inputs["raw_data"],
                     metadata=metadata,
                     sensor_ids=[
-                        "position"
-                    ],  # we don't care about other sensors
+                        sensor.sensor_id for sensor in metadata.sensors
```

Comment:
I believe we should load all sensors in the data, and the user can later filter out the ones they don't want.

Reply:
That sounds like a good addition! The data from the other sensors is not loaded into the frames: https://github.com/PySport/kloppy/blob/master/kloppy/infra/serializers/tracking/epts/serializer.py#L42-L59

We would need a way to store additional information per frame.